pkg/kubelet/prober/worker.go

```diff
@@ -152,9 +152,9 @@ func (w *worker) doProbe() (keepGoing bool) {
 	}
 
 	c, ok := api.GetContainerStatus(status.ContainerStatuses, w.container.Name)
-	if !ok {
+	if !ok || len(c.ContainerID) == 0 {
 		// Either the container has not been created yet, or it was deleted.
```

Comment:
If not, then I should probably change the comment too to remove the part about it being deleted.